test/timearray.jl

```diff
@@ -80,13 +80,30 @@ facts("conversion methods") do
     end
 end
 
+facts("index by integer works with both 1d and 2d time array") do
+
+    context("1d time array") do
+      @fact cl[1].timestamp --> [Date(2000,1,3)]
+      @fact cl[1].values    --> [111.94]
+      @fact cl[1].colnames  --> ["Close"]
+      @fact cl[1].meta      --> "AAPL"
+    end
+
+    context("2d time array") do
+      @fact ohlc[1].timestamp --> [Date(2000,1,3)]
+      @fact ohlc[1].values    --> [104.88 112.5 101.69 111.94]
+      @fact ohlc[1].colnames  --> ["Open", "High", "Low","Close"]
+      @fact ohlc[1].meta      --> "AAPL"
```

Comment:
Do you mean with `@fact` lines? I can fix that. I like the arrows to line up though within each `context` block, makes the tests much easier to read.